pandas/tests/tools/test_to_numeric.py

```diff
@@ -372,39 +417,6 @@ def test_str(data, exp, transform_assert_equal):
     assert_equal(result, expected)
 
 
-def test_datetime_like(tz_naive_fixture, transform_assert_equal):
```

Comment:
why are you removing these?

Reply:
Because I was changing the behavior of the function for date-like objects